libs/tokens/src/utils/validateTokenList.ts

```diff
@@ -0,0 +1,56 @@
+import type { TokenInfo, TokenList } from '@uniswap/token-lists'
+
+import type { Ajv, ValidateFunction } from 'ajv'
+
+enum ValidationSchema {
+  LIST = 'list',
+  TOKENS = 'tokens',
+}
+
+const validator = new Promise<Ajv>((resolve) => {
+  Promise.all([import('ajv'), import('@uniswap/token-lists/src/tokenlist.schema.json')]).then(([ajv, schema]) => {
```

Comment:
Disclaimer: I don't know what I'm talking about, just guessing.

Importing the types from `ajv` doesn't cause it to load sync regardless of loading it async here?

Reply:
>doesn't cause it to load sync regardless of loading it async here
It don't.
I haven't touched this file, just moved from legacy